reference/docs-conceptual/developer/ets/typeconverters.md

```diff
@@ -0,0 +1,110 @@
+| String                | RegEx             | Results of call to `Regx(valueToConvert)`.                                                                                                                          |
+| String                | Type              | Returns the appropriate type using the **valueToConvert** parameter to search **RunspaceConfiguration.Assemblies**.                                                 |
+| String                | Numeric           | If **valueToConvert** is "", returns `0` of the **resultType**. Otherwise the culture "culture invariant" is used to produce a numeric value.                       |
+| Integer               | System.Enum       | Converts the integer to the constant if the integer is defined by the enumeration. If the integer is not defined an **PSInvalidCastException** exception is thrown. |
+
+## Custom conversions
+
+If PowerShell cannot convert the type using a standard PowerShell language converter, it then checks
+for custom converters. PowerShell looks for several types of custom converters in the order
+described in this section. Be aware that references to the **valueToConvert** and **resultType**
+parameters refer to parameters of the `ConvertTo(System.Object,System.Type)` method. If a custom
+converter throws an exception, then no further attempt is made to convert the object and that
+exception is wrapped in a **PSInvalidCastException** exception which is then thrown.
+
+## PowerShell type converter
+
+PowerShell type converters are used to convert a single type or a family of types, such as all types
+that derive from the **System.Enum** class. To create a PowerShell type converter you must implement
+an PSTypeConverter class and associate that implementation with the target class. There are two ways
+of associating the PowerShell type converter with its target class.
+
+- Through the type configuration file
+- By applying the **TypeConverterAttribute** attribute to the target class
+
+PowerShell type converters, derived from the
+[PSTypeConverter](/dotnet/api/system.management.automation.pstypeconverter) abstract class, provide
+methods for converting an object to a specific type or from a specific type. If the
+**valueToConvert** parameter contains an object that has a PowerShell Type converter associated with
+it, PowerShell calls the
+`PSTypeConverter.ConvertTo(System.Object,System.Type,System.IFormatProvider,System.Boolean)` method
+of the associated converter to convert the object to the type specified by the **resultType**
+parameter. If the **resultType** parameter references a type that has a PowerShell type converter
+associated with it, PowerShell calls the
+`PSTypeConverter.ConvertFrom(System.Object,System.Type,System.IFormatProvider,System.Boolean)`
+method of the associated converter to convert the object from the type specified by the
+res****ultType parameter.
```

Comment:
What is that supposed to be, `res****ultType`?

Reply:
fixed